src/containerapp/azext_containerapp/_params.py

```diff
@@ -485,19 +485,26 @@ def load_arguments(self, _):
+        c.argument('replica_retry_limit', type=int, help='Maximum number of retries before the replica fails')
+        c.argument('replica_timeout', type=int, help='Maximum number of seconds a replica can execute')
+        c.argument('parallelism', type=int, help='Maximum number of replicas to run per execution')
         c.argument('workload_profile_name', options_list=['--workload-profile-name', '-w'], help='The friendly name for the workload profile')
-        c.argument('min_executions', type=int, help="Minimum number of job executions that are created for a trigger, default 0.")
-        c.argument('max_executions', type=int, help="Maximum number of job executions that are created for a trigger, default 100.")
-        c.argument('polling_interval', type=int, help="Interval to check each event source in seconds. Defaults to 30s.", default=30)
+        c.argument('min_executions', type=int, help="Minimum number of job executions that are created for a trigger")
+        c.argument('max_executions', type=int, help="Maximum number of job executions that are created for a trigger")
+        c.argument('polling_interval', type=int, help="Interval to check each event source in seconds.")
 
     with self.argument_context('containerapp job create') as c:
         c.argument('system_assigned', options_list=['--mi-system-assigned', c.deprecate(target='--system-assigned', redirect='--mi-system-assigned', hide=True)], help='Boolean indicating whether to assign system-assigned identity.', action='store_true')
         c.argument('trigger_type', help='Trigger type. Schedule | Event | Manual')
         c.argument('user_assigned', options_list=['--mi-user-assigned', c.deprecate(target='--user-assigned', redirect='--mi-user-assigned', hide=True)], nargs='+', help='Space-separated user identities to be assigned.')
+        c.argument('replica_completion_count', type=int, options_list=['--replica-completion-count', '--rcc'], help='Number of replicas that need to complete successfully for execution to succeed', default=1)
+        c.argument('replica_retry_limit', type=int, help='Maximum number of retries before the replica fails. Default: 0.', default=0)
+        c.argument('replica_timeout', type=int, help='Maximum number of seconds a replica can execute', default=1800)
+        c.argument('parallelism', type=int, help='Maximum number of replicas to run per execution', default=1)
+        c.argument('min_executions', type=int, help="Minimum number of job executions that are created for a trigger. Default: 0.", default=0)
+        c.argument('max_executions', type=int, help="Maximum number of job executions that are created for a trigger", default=100)
+        c.argument('polling_interval', type=int, help="Interval to check each event source in seconds.", default=30)
```

Comment:
May I ask if these params are new added and have been tested?

Reply:
Yes, these params are not new added, it added default value for `containerapp job create`. These arguments also used by `containerapp job update`, the update command no need the default value.